pkgs/applications/search/catfish/default.nix

```diff
@@ -3,13 +3,13 @@
 
 pythonPackages.buildPythonApplication rec {
   majorver = "1.4";
-  minorver = "2";
+  minorver = "4";
   version = "${majorver}.${minorver}";
   name = "catfish-${version}";
```

Comment:
Please get rif of the `name` here and only specify `pname`.

Reply:
What is the rationale behind that? I failed to find any documentation on it. Notice that `name` is used in the `src` attribute.

Reply:
It's not necessary anymore for `buildPython*`. Without name, it easier to override the attributes for using a custom version.

Reply:
And how should the url be written in this case? When I remove `name` and add `pname`, I get
```
error: undefined variable ‘name’ at /alt/nixpkgs/pkgs/applications/search/catfish/default.nix:11:84
```

Reply:
Use `${pname}-${version}`.

Reply:
Done.